packages/uicore/src/components/MultiTypeInput/MultiTypeInput.tsx

```diff
@@ -204,7 +204,7 @@ export function ExpressionAndRuntimeType<T = unknown>(props: ExpressionAndRuntim
       className={cx(mini ? css.mini : css.main, {
         [css.disabled]: type === MultiTypeInputType.RUNTIME || disabled
       })}
-      width={width}
+      width={width || '100%'}
```

Comment:
this can break at other places where width is not passed. the default width will be considered as 100%